packages/core/src/main/Account.ts

```diff
@@ -380,6 +384,10 @@ class Account extends EventEmitter {
           content.mentions = editedMentions;
         }
 
+        if (editedQuote) {
+          content.quote = editedQuote;
```

Comment:
Add comment that while this is possible on protobuf level it should be ignored on receiver side according to spec

Reply:
I added it in `EditedTextContent.ts`.